velox/dwio/dwrf/reader/BinaryStreamReader.cpp

```diff
@@ -97,9 +97,9 @@ std::unique_ptr<BinaryStripeStreams> BinaryStreamReader::next() {
       stripeReaderBase_, columnSelector_, stripeIndex_++);
 }
 
-std::unordered_map<uint32_t, proto::ColumnStatistics>
+std::unordered_map<uint32_t, ColumnStatisticsWrapper>
```

Comment:
We have some problem here, the wrapper is just a reference but here we are returning actual values.  If ORC does not need it, can we keep the signature and make it DWRF only (with a check)?